heartbeat/docs/overview.asciidoc

```diff
@@ -0,0 +1,32 @@
+[[heartbeat-overview]]
+== Overview
+
+//TODO: Add Beta tag, when available, to generate the labels indicating that this is a Beta feature.
+
+Heartbeat is a lightweight daemon that you install on a remote server
+to ping your services periodically and determine whether they are available.
+Unlike {metricbeat}/index.html[Metricbeat], which only tells you if your servers
+are up or down, Heartbeat tells you whether your services are reachable. If
```

Comment:
2nd paragraph should probably  appear here and then be followed by the bit about load balancing.